virocon/distributions.py

```diff
@@ -325,7 +562,32 @@ def _fit_lsq(self, data):
         
 class LogNormalNormFitDistribution(LogNormalDistribution):
     #https://en.wikipedia.org/wiki/Log-normal_distribution#Estimation_of_parameters
+    """
+    A Lognormal Distribution. 
+    
+    :math:`f(x) = \\frac{1}{x\\widetilde{\\sigma} \\sqrt{2\\pi}}\\exp \\left[ \\frac{-(\\ln x - \\widetilde{\\mu})^2}{2\\widetilde{\\sigma}^2}\\right]`
+     
+    
+    Parameters
+    ----------
+    mu : float
+        Mean parameter of the lognormal distribution. Defaults to 0.
```

Comment:
See LogNormalDistribution, but LogNormalNormFitDistribution will be (re)moved anyway.